recipes/hidapi/all/conanfile.py

```diff
@@ -99,13 +109,11 @@ def package(self):
 
     def package_info(self):
         if self.settings.os in ["Linux", "FreeBSD"]:
-            self.cpp_info.components["libusb"].names["pkg_config"] = "hidapi-libusb"
             self.cpp_info.components["libusb"].set_property("pkg_config_name", "hidapi-libusb")
             self.cpp_info.components["libusb"].libs = ["hidapi-libusb"]
             self.cpp_info.components["libusb"].requires = ["libusb::libusb"]
             self.cpp_info.components["libusb"].system_libs = ["pthread", "dl", "rt"]
 
-            self.cpp_info.components["hidraw"].names["pkg_config"] = "hidapi-hidraw"
             self.cpp_info.components["hidraw"].set_property("pkg_config_name", "hidapi-hidraw")
```

Comment:
Ok. This recipe needs more love. This recipe should rely on CMake since 0.11.0.

https://github.com/libusb/hidapi/releases/tag/hidapi-0.11.0:

> NOTE: Starting with this release Autotools build scripts for HIDAPI are deprecated and going to be obsolete in near future.
> All users of HIDAPI should switch to CMake build scripts instead.